README.md

```diff
@@ -2,58 +2,146 @@
-helm install --namespace linkerd -n linkerd giantswarm-playground-catalog/linkerd2-app
+We recommend deploying the app by applying an `App` CR (Custom Resource) onto your management cluster. You can use the [`kubectl gs`](https://docs.giantswarm.io/ui-api/kubectl-gs/) plugin to generate a valid `App` CR with command
+
+```
+kubectl gs template app \
+  --catalog giantswarm \
+  --name linkerd2-app \
+  --namespace linkerd \
+  --cluster <your-cluster-id>  \
+  --version 0.6.0 \
+  --user-configmap my-linkerd-values.yaml \
+  --user-secret my-linkerd-certificates.yaml \
+  --namespace-labels "linkerd.io/is-control-plane=true,config.linkerd.io/admission-webhooks=disabled,linkerd.io/control-plane-ns=linkerd" \
+  --namespace-annotations "linkerd.io/inject=disabled"
 ```
 
-## Configuration
+The final `App` CR should look like this:
+
+```yaml
+...
+apiVersion: application.giantswarm.io/v1alpha1
+kind: App
+metadata:
+  name: linkerd2-app
+  namespace: <your-cluster-id>
+spec:
+  catalog: giantswarm
+  kubeConfig:
+    inCluster: false
+  name: linkerd2-app
+  namespace: linkerd
+  namespaceConfig:
+    annotations:
+      linkerd.io/inject: disabled
+    labels:
+      config.linkerd.io/admission-webhooks: disabled
+      linkerd.io/control-plane-ns: linkerd
+      linkerd.io/is-control-plane: "true"
+  userConfig:
+    configMap:
+      name: linkerd2-app-userconfig-<your-cluster-id>
+      namespace: <your-cluster-id>
+    secret:
+      name: linkerd2-app-userconfig-<your-cluster-id>
+      namespace: <your-cluster-id>
+  version: 0.6.0
+
+```
+
+When installing via the Giant Swarm web UI, you'll need to apply the above labels and annotations manually.
+
+### Step 4: After deployment
+
+- Disable proxy injections for pods running in the `kube-system` namespace by applying certain labels and annotations
+
+```bash
+kubectl annotate namespace kube-system linkerd.io/inject=disabled
+kubectl label namespace kube-system config.linkerd.io/admission-webhooks=disabled
+```
+
+- Optional but recommended: You can use the `linkerd` cli as usual with this app as we're using the default namespaces. (`linkerd` and `linkerd-cni`). You can download it from the [linkerd release page](https://github.com/linkerd/linkerd2/releases/tag/stable-2.10.2).
+
+- We strongly recommend installing the `linkerd viz` extension using the [`linkerd`](#usage-with-linkerd-cli) by executing
+
+```bash
+linkerd viz install | kubectl apply -f -
+```
+
+After installation, you can open the dashboard by executing
+
+```
+linkerd viz dashboard
+```
+
+## Mesh your workloads
+
+After installation, linkerd looks for a `linkerd.io/inject: enabled` annotation on `Namespaces` or other workload resources. Adding this annotation to your workload namespaces will trigger automatic proxy container injection to your pods. More information on proxy injection can be found on the ["Automatic Proxy Injection" page](https://linkerd.io/2.10/features/proxy-injection/) in the upstream documentation.
```

Comment:
Maybe let's add info about how to use `namespaceConfiguration` in App CRs to get the same effect for apps deployed with app-platform?

Reply:
added :heavy_check_mark: